Neos.Flow/Classes/Mvc/Routing/Router.php

```diff
@@ -265,6 +267,14 @@ protected function createRoutesFromConfiguration()
                 }
                 $routesWithHttpMethodConstraints[$uriPattern] = false;
             }
+            if (isset($routeConfiguration['cache'])) {
+                if (isset($routeConfiguration['cache']['lifetime']) && is_int($routeConfiguration['cache']['lifetime'])) {
+                    $route->setCacheLifetime(RouteLifetime::createFromInt($routeConfiguration['cache']['lifetime']));
+                }
+                if (isset($routeConfiguration['cache']['tags']) && is_array($routeConfiguration['cache']['tags'])) {
```

Comment:
same here
```suggestion
                if (isset($routeConfiguration['cache']['tags'])) {
                    assert(is_array($routeConfiguration['cache']['tags']), 'cache.tags must be of type array');
```

Reply:
This was just meant as an example.
My point was not to silently ignore invalid configuration, you could replace the `assert` with an exception of course.